auth_server/authn/ldap_auth.go

```diff
@@ -173,7 +179,9 @@ func (la *LDAPAuth) ldapSearch(l *ldap.Conn, baseDN *string, filter *string, att
 		return "", err
 	}
 
-	if len(sr.Entries) != 1 {
+	if len(sr.Entries) == 0 {
+		return "", nil // User does not exist
+	} else if len(sr.Entries) > 1 {
 		return "", fmt.Errorf("User does not exist or too many entries returned.")
```

Comment:
In this case, i think we need to change the error message : "User does not exist or too many entries returned." To "Too many entries returned."